src/inference/src/cpp_interfaces/interface/ie_iinfer_request_internal.cpp

```diff
@@ -157,6 +159,175 @@ void IInferRequestInternal::SetBlob(const std::string& name, const Blob::Ptr& us
     }
 }
 
+void IInferRequestInternal::SetBlobs(const std::string& name, const std::vector<Blob::Ptr>& blobs) {
+    bool all_memory = std::all_of(blobs.begin(), blobs.end(), [](const Blob::Ptr& item) {
+        return item && item->is<MemoryBlob>() && !item->is<RemoteBlob>();
```

Comment:
so any default impl for BatchedBlob of remote blobs is up to the plugins?

Reply:
yes